src/cmd/dev.go

```diff
@@ -23,17 +23,44 @@ import (
 	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
 	"github.com/mholt/archiver/v3"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 var extractPath string
 
-var prepareCmd = &cobra.Command{
-	Use:     "prepare",
-	Aliases: []string{"prep"},
-	Short:   lang.CmdPrepareShort,
+var devCmd = &cobra.Command{
+	Use:     "dev",
+	Aliases: []string{"prepare"},
+	Short:   lang.CmdDevShort,
 }
 
-var prepareTransformGitLinks = &cobra.Command{
+var devDeployCmd = &cobra.Command{
+	Use:   "deploy",
+	Args:  cobra.ExactArgs(1),
+	Short: lang.CmdDevDeployShort,
+	Long: lang.CmdDevDeployLong,
+	Run: func(cmd *cobra.Command, args []string) {
+		pkgConfig.CreateOpts.BaseDir = args[0]
```

Comment:
```suggestion
		if len(args) > 0 {
			pkgConfig.CreateOpts.BaseDir = args[0]
		} else {
			var err error
			pkgConfig.CreateOpts.BaseDir, err = os.Getwd()
			if err != nil {
				message.Fatalf(err, lang.CmdPackageCreateErr, err.Error())
			}
		}
```

This could also use @AustinAbro321 's function he added in #2171